docs/flink-getting-started.md

```diff
@@ -83,7 +83,12 @@ Start the Flink SQL client. There is a separate `flink-runtime` module in the Ic
 # HADOOP_HOME is your hadoop root directory after unpack the binary package.
 export HADOOP_CLASSPATH=`$HADOOP_HOME/bin/hadoop classpath`   
 
+# Below works for 1.15 or less
 ./bin/sql-client.sh embedded -j <flink-runtime-directory>/iceberg-flink-runtime-1.16-{{% icebergVersion %}}.jar shell
```

Comment:
Since it works for 1.15 or less, should change it to `iceberg-flink-runtime-1.15-`? Or use a version variable instead?

Reply:
good catch. fixed